editor.html

```diff
@@ -87,12 +87,18 @@ <h2 class="modal-title"><i class="fa fa-upload"></i> <strong>{{ load-title }}</s
                 <input type="file" style="display: none" name="load-form-file-upload" id="file-upload-input">
                 <p>{{ instructions }}<br><a href="#" id="file-upload-link" class="load-drag-target load-toggle action">{{ toggle-file }}</a></p>
             </div>
-            <h2 class="modal-title"><i class="fa fa-download"></i> <strong>{{ save-title }}</strong></h2>
+            <h2 class="modal-title"><i class="fa fa-download"></i> <strong>{{ save-but }}</strong></h2>
             <div class="save-buttons-container">
                 <button type="button" class="action save-button py" id="save-py" title="{{ save-py }}"><i class="fa fa-download"></i> {{ save-py}}</button>
                 <button type="button" class="action save-button hex" id="save-hex" title="{{ save-hex }}"><i class="fa fa-download"></i> {{ save-hex}}</button>
             </div>
-            <h2 class="modal-title"><i class="fa fa-download"></i> <strong>{{ program-title }} {{ fs-title }}</strong></h2>
+            <div id="addFile">
+              <div id="addFileHeader">
+                <h2 class="modal-title"><i class="fa fa-download"></i> <strong>{{ files-title }}</strong></h2>
+              </div>
+              <div id="addFileHelp"><a id="expandHelpPara"><i class="fa fa-info-circle" aria-hidden="true"> <span>{{ help-button }}</span></i></a></div>
+            </div>
+            <div id="fileHelpPara">{{ file-help-text }}<a href='help.html#fs' target="_blank">{{ help-link }}</a></div>
```

Comment:
The HTML attributes should use double quotes.

```suggestion
            <div id="fileHelpPara">{{ file-help-text }}<a href="help.html#fs" target="_blank">{{ help-link }}</a>.</div>
```

Also, we should have a period at the end of the sentence.